source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -0,0 +1,190 @@
+documents. A replace operation specifies the fields and values to replace
+a single document from your collection. 
+
+In the following examples, a paint store sells five different
+colors of paint. The ``paint_inventory`` collection represents their
+current inventory: 
+
+.. code-block:: json
+
+    { "_id": 1, "color": "red", "qty": 5 }
+    { "_id": 2, "color": "purple", "qty": 8 }
+    { "_id": 3, "color": "yellow", "qty": 0 }
+    { "_id": 4, "color": "green", "qty": 6 }
+    { "_id": 5, "color": "pink", "qty": 0 }
+
+.. _update-operation:
+
+Update
+------
+
+Update operations can change the field and value. They apply changes
+specified in an update document to one or more documents that match your
+query filter. 
+
+The :java-docs:`updateOne() <apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#updateOne(org.bson.conversions.Bson,org.bson.conversions.Bson)>` 
+method changes the first document your query filter matches and the
+:java-docs:`updateMany() <apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#updateMany(org.bson.conversions.Bson,org.bson.conversions.Bson)>` 
+method changes all the documents your query filter matches. 
+
+You can call the ``updateOne()`` and ``updateMany()`` methods on a
+``MongoCollection`` instance as follows: 
+
+.. code-block:: java
+
+    collection.updateOne(query, updateDocument);
+
+    collection.updateMany(query, updateDocument);
+    
+Update Operation Parameters:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- ``query`` specifies the document you want to update in your collection
+- ``updateDocument`` specifies the field and value to change in the matching document or documents. For this example, we use the :doc:`Updates builder </fundamentals/builders/updates>` to create the update document.
+
+You can create the ``updateDocument`` using an ``Updates`` builder as
+follows:
+
+.. code-block:: java
+
+    Bson updateDocument = Updates.<update operator>("<field>", "<value>");
+
+See the MongoDB API documentation for a :java-core-api:`complete list of
+update operators and their usage <com/mongodb/client/model/Updates.html>`.
+
+Example
+~~~~~~~
+
+The paint store receives a fresh shipment and needs to update their inventory.
+The shipment contains 20 cans of each paint color.
+
+To update the inventory, call the ``updateMany()`` method specifying the
+following:
+
+- A query filter that matches all the colors 
+- An update document that contains instructions to increment the ``qty`` field by "20"
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Update.java
+   :language: java
+   :dedent:
+   :start-after: begin updateManyExample
+   :end-before: end updateManyExample
+
+The method returns:
+
+.. code-block:: json   
+
+    AcknowledgedUpdateResult{ matchedCount=5, modifiedCount=5, upsertedId=null } 
+
+The ``AcknowledgedUpdateResult`` tells us that five documents matched
+the query, five documents got modified and there was no upsert.
```

Comment:
I think this sentence is passive voice. You can make it active voice by specifying what matched the query and what did the modifying.